Makefile.include

```diff
@@ -173,6 +173,27 @@ include $(RIOTMAKE)/boards.inc.mk
 # Debug targets for build system migration
 include $(RIOTMAKE)/dependencies_debug.inc.mk
 
+# Use TOOLCHAIN environment variable to select the toolchain to use.
+# If native, TOOLCHAIN for OSX is llvm
+ifeq ($(BOARD),native)
+  ifeq ($(OS),Darwin)
+    TOOLCHAIN ?= llvm
+  endif
+endif
+# Use override so that we can redefine a variable set on the command line (as
+# opposed to one set in the environment)
+ifeq (clang,$(TOOLCHAIN))
+  # TOOLCHAIN = clang is an alias for TOOLCHAIN = llvm
+  override TOOLCHAIN := llvm
+endif
+ifeq (gcc,$(TOOLCHAIN))
+  # TOOLCHAIN = gcc is an alias for TOOLCHAIN = gnu
+  override TOOLCHAIN := gnu
```

Comment:
Does this means that TOOLCHAIN is equal to `gnu` when Makefile.dep is processed ? If yes, the toolchain specific dependency file should be called `Makefile.gnu.dep` and not `Makefile.gcc.dep` (in  examples/gnrc_minimal) ?

Reply:
True!